packages/changelog-extract/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@brisk-docs/changelog-extract",
+  "version": "1.0.0",
+  "description": "This package is used for extracting changelogs from your projects",
+  "main": "index.js",
+  "repository": " git@bitbucket.org:atlassian/changelog-extract.git",
```

Comment:
The repository info should be updated for its new home (though we may move it again soon)